packages/orbit-components/src/Tag/index.tsx

```diff
@@ -80,25 +79,33 @@ const Tag = React.forwardRef<HTMLDivElement, Props>(
           ],
         )}
         data-test={dataTest}
-        id={id}
-        ref={ref}
-        onClick={onClick}
-        // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
-        tabIndex={(onClick || onRemove) && 0}
-        role={onClick || onRemove ? "button" : undefined}
-        onKeyDown={ev => buttonClickEmulation(ev, onClick)}
       >
-        {iconLeft && (
-          <div className="pe-200 [&_svg]:size-icon-small flex flex-row items-center justify-center">
-            {iconLeft}
-          </div>
-        )}
-        {children}
+        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+        <div
+          className={cx(
+            "orbit-tag flex items-center",
+            onRemove ? "ps-200 py-200 pe-100 rounded-l-150" : "p-200 rounded-150",
+          )}
+          id={id}
+          ref={ref}
+          onClick={onClick}
+          // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+          tabIndex={(onClick || onRemove) && 0}
+          role={onClick || onRemove ? "button" : undefined}
+          onKeyDown={ev => buttonClickEmulation(ev, onClick)}
+        >
+          {iconLeft && (
+            <div className="pe-200 [&_svg]:size-icon-small flex flex-row items-center justify-center">
+              {iconLeft}
+            </div>
+          )}
+          {children}
+        </div>
         {onRemove && (
           <div
             className={cx(
               "orbit-tag-close-container",
-              "ms-200 flex rounded-full",
+              "pe-200 ps-100 rounded-r-150 flex items-center justify-center",
```

Comment:
I added `ps-100` to make space between outline and close icon when it's focused -> visual reasons. Same for "title" part of the tag, when `removable=true`.

`ps-0`
<img width="147" alt="Snímek obrazovky 2025-03-07 v 19 24 43" src="https://github.com/user-attachments/assets/77c7f85e-c385-43ec-be97-30d2de8bdc43" />


`ps-100`
<img width="149" alt="Snímek obrazovky 2025-03-07 v 19 24 21" src="https://github.com/user-attachments/assets/5a2e6ffa-7a64-4c62-8ffe-8d055a9e5e55" />

Reply:
It is very weird that the user is able to click outside of the icon and trigger the onRemove. Can you please check how other design systems approach this? 😕

Reply:
My first idea was to have "clickable" only the `X` icon, but then I was thinking about it, and it could be difficult to click that area on mobile phones. also, it can easily happen that the user will click `onClick` function instead of `onRemove`, because there won't be enough space. 

 I've found some articles about "minimum" target size:
- https://dequeuniversity.com/rules/axe/4.9/target-size
- https://tetralogical.com/blog/2022/12/20/foundations-target-size/

Here is an example where the "dismiss" button is 100% height of the tag. The design is quite different, but it correspond the minimum 24px size according the accessibility rules.
https://carbondesignsystem.com/components/tag/code/

Reply:
Alright, I am sold. But I would **loudly** present this to designers so they can think of a solution

Reply:
I'll write a note to the _accessibility compliance_ doc.

Also, I can see that `onRemove` and `onClick` props are used in search - check [sourcebot](https://sourcebot.skypicker.com/search?query=orbit-components%2Flib%2FTag%20repo%3Afrontend%2F*%20-repo%3A%5Egithub%5C.com%2Fkiwicom%2Forbit%24), however, they put the same function in both props. so the functionality is the same for both props, they just wanted to have a cross button in the Tag component (from what I understand).

https://sourcebot.skypicker.com/search?query=orbit-components%2Flib%2FTag%20repo%3Afrontend%2F*%20-repo%3A%5Egithub%5C.com%2Fkiwicom%2Forbit%24

@DSil

Reply:
> they put the same function in both props. so the functionality is the same for both props, they just wanted to have a cross button in the Tag component

Yes, but that is not an assumption we can make for all the cases